tests/e2e/test_jenkins.py

```diff
@@ -16,13 +16,20 @@
 import sys
 
 from cibyl.cli.main import main
-from tests.e2e.fixture import JenkinsTest
+from tests.e2e.fixtures import JenkinsTest
 
 
 class TestJenkins(JenkinsTest):
     """Tests queries regarding the Jenkins source.
     """
 
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.add_job('test_1')
```

Comment:
If I try to run this test I get a 403 error:
`requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: http://localhost:8080/createItem?name=test_1`
Opening the url in the browser I get

```This URL requires POST

The URL you're trying to access requires that requests be sent using POST (like a form submission). The button below allows you to retry accessing this URL using POST. URL being accessed:

http://localhost:8081/createItem?name=test_1

If you were sent here from an untrusted source, please proceed with caution.```

Reply:
You may need to delete the image so that the test can rebuild it. I had to make a change to the Dockerfile. It is usually named: jenkins_jenkins_1.

Reply:
Oh makes sense, I'll try

Reply:
You were right, after rebuilding the image it works, without any issue, thanks!